index.html

```diff
@@ -0,0 +1,362 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+  <meta charset="utf-8">
+  <title></title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="icon" href="imgs/favicon.png">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+
+  <link rel="stylesheet" href="css/style.css">
+
+  <meta name="theme-color" content="#fafafa">
+</head>
+
+<body>
+  <header>
+  	<div id="left-header">
+  	 <button id="main-nav">
+  	 	<i class="fas fa-bars"></i>
+  
+  	 </button>
+  	 <div id="logo-container">
+  	 	<a href="https://youtube.com/">
+  	 		<img id="logo" src="imgs/logo.png" title="youtube logo" alt="youtube logo">
+  	 	</a>
+  	 </div>
+  	</div>
+    <!-- end of left-header -->
+
+  	<div id="search-form-container">
+  		<form>
+  			<input type="text" placeholder="Search">
+  			<button>
+  				<i class="fas fa-search"></i>
+
+  			</button>
+  		</form>
+
+  	</div>
+    <!-- end of search form container -->
+  	<div id="options-container">
+
+  		<button id="cam">
+  			<svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope yt-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope yt-icon">
+        <path d="M17 10.5V7c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1v10c0 .55.45 1 1 1h12c.55 0 1-.45 1-1v-3.5l4 4v-11l-4 4zM14 13h-3v3H9v-3H6v-2h3V8h2v3h3v2z" class="style-scope yt-icon"></path>
+      </g></svg>
+  		</button>
+
+      	<button id="grid-icon">
+      	</button>
+
+      	<button id="comments-icon">
+      		<i class="far fa-comment-dots"></i>
+      	</button>
+
+      	<button id="bill-icon">
+      		<i class="far fa-bell"></i>
+      	</button>
+      	<button id="user-icon">
+      		<span>A</span>
+      	</button>
+
+  	</div>
+    <!-- end of options container -->
+  </header>
+  <div class="container">
+    <main>
+      
+        <iframe width="853" height="480" src="https://www.youtube.com/embed/V74l_zS1x8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
```

Comment:
The frameborder attribute on the iframe element is obsolete. Use CSS style instead.